src/test/end-to-end/pages/caseDetails/solicitorApplyProbate/copiesTabConfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "tabName": "Copies",
+    "fields": [
+      "How many extra UK copies of the grant do you need?",
+      "How many sealed and certified copies of the grant do you need for use outside the UK?",
+      "How do you want to pay?"
+    ],
+    "dataKeys": [
+      "page4_extraCopiesUK",
+      "page4_outsideUKGrantCopies",
+      "page5_paymentType"
+    ]
+  }
+  
```

Comment:
very minor issue - if you add a carriage return at end of line - this complaining symbol will go. Most lint rules do requrie a carriage return at end of line